internal/website/content/howto/sql/_index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "MySQL/PostgreSQL"
+date: 2019-06-21T09:26:56-07:00
+showInSidenav: true
+---
+
+Connecting to cloud providers' hosted database services requires additional
+steps to ensure the security of the connection. For example, each of AWS,
+GCP, and Azure require the use of custom certificate authorities to be
+configured in the client. GCP requires a custom proxy with authentication
+credentials. The Go CDK makes opening these connections easier while still
```

Comment:
Micro-nit: consider moving "The Go CDK..." to its own paragraph since it's logically separate. @cnguyen83's opinion here should be useful